### ISSUE TITLE
Dualshock PS button (Home button) in-game

### ISSUE THREAD

**Issue by backslash-f** [Plnda/Gradia#2]
I noticed that the PS button opens the side-menu at the right of the screen in Stadia -- but only if there is no game running. 

I tested with Doom and Destiny 2 and it seems that the "home button" is lost in-game?

It's weird because it's indeed mapped `GamePadButton(pressed: buttonHome.isPressed, value: Double(buttonHome.value))`

¯\_(ツ)_/¯

**Comment by Plnda** [comment]
Yes this is something i can't fix right now. iOS snags that button by itself. I might do some more debugging 👍
